src/todo/todo.controller.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  Controller,
+  Get,
+  Post,
+  Body,
+  Patch,
+  Param,
+  Delete,
+} from '@nestjs/common';
+import { TodoService } from './todo.service';
+import { CreateTodoDto } from './dto/create-todo.dto';
+import { UpdateTodoDto } from './dto/update-todo.dto';
+
+@Controller('todo')
+export class TodoController {
```

Comment:
todos should belong to authorized user and all manipulations with todos should be done for authorized user too